libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -40,6 +40,16 @@ import kotlinx.coroutines.channels.ReceiveChannel
 class MapboxNavigation(
     private val context: Context,
     private val accessToken: String?,
+    private val navigationOptions: NavigationOptions = NavigationOptions.Builder(
+        TWELVE_HOURS,
+        ROUNDING_INCREMENT_FIFTY,
+        distanceFormatter = MapboxDistanceFormatter(
+            context.applicationContext,
+            "ja",
+            METRIC,
+            ROUNDING_INCREMENT_FIFTY
+        )
+    ).build(),
```

Comment:
Can we return this default builder in a static method? Like `MapboxNavigation#defaultNavigationOptions(context)` so that users can fetch that reference and change just a handful of options instead of being forced to always compose from scratch?

Reply:
Yeah, we should expose builders for all of the options https://github.com/mapbox/mapbox-java/issues/1108#issuecomment-577309659

Reply:
@LukasPaczos  Let's put default navigation options inside NavigationOptions instead of MapboxNavigation. What do you think?

Reply:
We can't because `MapboxDistanceFormatter` and possibly other implementation classes in the future are exposed from the core module.